llvm/lib/Target/AMDGPU/lds-dma-waits.ll

```diff
@@ -0,0 +1,154 @@
+; RUN: llc -march=amdgcn -mcpu=gfx900 < %s | FileCheck %s --check-prefixeses=GCN,GFX9
+; RUN: llc -march=amdgcn -mcpu=gfx1030 < %s | FileCheck %s --check-prefixeses=GCN,GFX10
```

Comment:
--check-prefixeses  -> --check-prefixes

Reply:
Fixed. Also moved test to a proper place.